src/account/default.ts

```diff
@@ -249,6 +251,162 @@ export class Account extends Provider implements AccountInterface {
     return this.estimateInvokeFee(calls, transactionsDetail);
   }
 
+  public async estimateFeeBulk(
+    transactions: Array<TransactionBulk>,
+    { nonce: providedNonce, blockIdentifier }: EstimateFeeDetails = {}
+  ): Promise<Array<EstimateFee>> {
+    const nonce = toBN(providedNonce ?? (await this.getNonce()));
+    const version = toBN(feeTransactionVersion);
+    const chainId = await this.getChainId();
+
+    const invocations: any = await Promise.all(
```

Comment:
@irisdv this is a big code dup with the specialized `estimate*Fee` functions, please consider some refactoring so each flavor is managed in a single place in the code and the wrapping functions use that place instead of implementing it on their own